types/powermeter.yaml

```diff
@@ -299,3 +299,10 @@ types:
           This is intended for meters that only support signing a collection of meter values.
         type: object
         $ref: /units_signed#/SignedMeterValue
+      temperature1:
```

Comment:
This looks to LEM specific to me. Not sure which temperatures it actually measures, but would it make sense to you to add more generic properties like:
* ambient_temperature
* internal_temperature
* conductor_temperature
* ...

Reply:
I agree but since this is not defined by all the power meters, this has to cover everyone. 
What if I have 2 cable temps? Or a high and low temp? 
Starting to give them "actual functional" names might pollute the interface with a lot of unused variables.
I spoke with Cornelius as well and he proposed to use temp 1 and 2 and their meaning is Powermeter dependent. Since anyway it is - some have it, some have it not, some measure the ambient some measure the internal some measure the high and the low. 
You get the picture. 
So I would be in the favor of keeping them temp 1 and 2 and their meaning is Powermeter actual implementation dependent.

Reply:
If it's really that opaque we could also just add an array of temperature values here, what if there's a third sensor for example :D

Reply:
> I agree but since this is not defined by all the power meters, this has to cover everyone.
> Starting to give them "actual functional" names might pollute the interface with a lot of unused variables

For this we have optional properties. There are a lot of properties in the powermeter already defined this way.

> What if I have 2 cable temps? Or a high and low temp?

As @hikinggrass pointed out, what if we have 3? That is indeed specific to the powermeter and should therefore not be covered in a generic interface like powermeter like this. An interface is meant to be required by other modules. How should other modules know what temp1 or temp2 or temp3 are, especially if they mean different things for different implementations. Defining it this way doesn't seem useful to me.

If we add it like this, Id agree with @hikinggrass and add an array of temperature values, but I would rather add more meaningful properties as described above, maybe defining a type for a temperature and adding optional low / high values to it as properties.

Reply:
I think an array is a good idea